decentraland/unity-renderer
# fix: smart item compatibility broken the scene

## What does this PR change?

This PR fixes a very specific bug Fixes #813 where if you transform on a smart item entity in the builder in world and without deselecting the item and then you try to exit the compatibility between builder and builder in world breaks

## How to test the changes?

1. Open an scene in builder
2. Add an smart item and publish the scene in the world
2. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/biw-smart-item&ENABLE_BUILDER_IN_WORLD
3. Open the same scene that you modified in builder in world ( going to the same land where you published)
4. Move the smart item that you placed and DO NOT DESELECT IT
5. Try to exit from builder in world
6. Check that the scene can be opened in the builder

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
